### ISSUE TITLE
Document system requirements for `cellxgene[prepare]`

### ISSUE THREAD

**Issue by mckinsel** [chanzuckerberg/cellxgene#464]
I think `pip install cellxgene[louvain]` isn't going to work for a lot of people. On ubuntu you need

```bash
apt-get install build-essential libxml2-dev python3-dev zlib1g-dev
```
and on centos
```bash
yum install gcc gcc-c++ libxml2-devel make python36u-devel zlib-devel
```
(or however you install python headers)

**Comment by csweaver** [comment]
I'll bring this use-case up when we have our OSS training this week

**Comment by sidneymbell** [comment]
Fixed via #1030
